plugins/extractors/postgres/README.md

```diff
@@ -6,25 +6,23 @@
 source:
   type: postgres
   config:
-    host: localhost:5432
-    user_id: admin
-    password: 1234
+    connection_url: postgres://admin:pass123@localhost:3306/testDB?sslmode=disable
+    database_name: testDB
```

Comment:
btw, is `database_name` here really needed?

Reply:
Removing it, forgot to remove it after changes